cmd/telegraf/telegraf.go

```diff
@@ -261,6 +259,7 @@ func runAgent(ctx context.Context,
 
 	logger.SetupLogging(logConfig)
```

Comment:
It would be nice to get the "starting telegraf" message to appear in log files. Moving it after the agent is already started makes it lose some value, as telegraf has already started by the time it prints.

Initializing logging does require the config to be read but doesn't require the agent to be created. (everything in LogConfig comes from ag.Config which is the same as c passed into NewAgent()

Maybe as a compromise we should log "starting" after the config has been read but before we call agent.NewAgent()?